roles/matrix-bot-buscarron/defaults/main.yml

```diff
@@ -76,6 +76,9 @@ matrix_bot_buscarron_homeserver: "{{ matrix_homeserver_container_url }}"
 # forms configuration
 matrix_bot_buscarron_forms: []
 
+# Disable encryption
+matrix_bot_buscarron_noencryption:
```

Comment:
Setting an explicit value here (e.g. `false`, `0`) will make this more clear what this variable needs to contain, without people having to go and dig into the Buscarron documentation.

Alternatively, we can leave it as `null`, but have 1 more comment line that tells you how to make use of this variable.

Reply:
Will do with the next release